src/test/java/com/thebois/models/world/StructureTest.java

```diff
@@ -0,0 +1,40 @@
+package com.thebois.models.world;
+
+import org.junit.jupiter.api.Test;
+
+import com.thebois.models.Position;
+import com.thebois.models.world.structures.Structure;
+import com.thebois.models.world.structures.StructureType;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class StructureTest {
+
+    @Test
+    public void getPositionTest() {
+
+        // Arrange
+        final Position position = new Position(123, 456);
+        final Structure structure = new Structure(123, 456);
+
+        // Act
+        final Position structurePosition = structure.getPosition();
+
+        // Assert
+        assertThat(structurePosition).isEqualTo(position);
+    }
+
+    @Test
+    public void getTypeTest() {
+
+        // Arrange
+        final Position position = new Position(123, 456);
+
+        // Act
+        final Structure structure = new Structure(position, StructureType.HOUSE);
```

Comment:
Not sure this is a part of Act since it just sets up a structure, we don't act on the object.

Reply:
You're probably right, I'll just remove the act commet.

Reply:
On the other hand, what is being tested here is that the constructor sets the value correctly. So in this case you could argue that *constructing* the object is the *acting*.